src/twitter.ts

```diff
@@ -31,6 +31,7 @@ export interface Tweet {
   in_reply_to_user_id_str?: string
   in_reply_to_screen_name?: string
   user: User
+  retweeted_status?: Tweet
```

Comment:
It would be better to use `retweeted` boolean flag instead of `retweeted_status`.
https://developer.twitter.com/en/docs/tweets/data-dictionary/overview/tweet-object